dali/operators/image/resize/resize_attr.cc

```diff
@@ -22,66 +22,85 @@
+
+Dimensions with 0 extent are treated as absent and the output size will be calculated
+based on other extents and ``mode`` argument.)code", {}, true)
+  .AddOptionalArg("mode", R"code(Resize mode.
+
+Here is a list of the supported values:
+
+* | ``"default"`` - image is resized to the specified size.
+  | Missing extents are scaled with the average scale of the provided ones extents.
+* | ``"stretch"`` - image is resized to the specified size.
+  | Missing extents are not at all scaled.
+* | ``"not_larger"`` - image is resized, keeping the aspect ratio, so that no extent of the
+    output image exceeds the specified size.
+  | For example, a 1280x720, with a desired output size of 640x480, actually produces
+    a 640x360 output.
+* | ``"not_smaller"`` - image is resized, keeping the aspect ratio, so that no extent of the
+    output image is smaller than specified.
+  | For example a 640x480 imag,e with a desired output size of 1920x1080, actually produces
+    a 1920x1440 output.
+
+This argument is mutually exclusive with ``resize_longer`` and ``resize_shorter``)code", "default")
+  .AddOptionalArg("resize_shorter", R"code(The length of the shorter dimension of the resized image.
+
+This option is mutually exclusive with ``resize_longer`` and explicit size arguments, and
+the operator keeps the aspect ratio of the original image. This option is equivalent to specifying
+the same size for all dimensions and ``mode="not_smaller"``. The longer dimension can be bounded
+by setting the ``max_size`` argument. See ``max_size`` argument documentation for
+more information.)code", 0.f, true)
+  .AddOptionalArg("resize_longer", R"code(The length of the longer dimension of the resized image.
+
+This option is mutually exclusive with ``resize_shorter`` and explicit size arguments, and
+the operator keeps the aspect ratio of the original image. This option is equivalent to specifying
+the same size for all dimensions and ``mode="not_larger"``.
+)code", 0.f, true)
+  .AddOptionalArg<vector<float>>("max_size", R"code(Limit of the output size.
+
+When resizing uses the ``resize_shorter``, but not the"not_smaller", mode or leaving some extents
+unspecified, some images with high aspect ratios might produce extremely large outputs.
+
+This parameter puts a limit to how big the output can become. This value can be specified per-axis
+or uniformly for all axes.)code", {}, false)
+  .AddOptionalArg("subpixel_scale", R"code(If ``True``, fractional sizes, directly
+specified or calculated, will cause the input ROI to be adjusted to keep the scale factor.
+
+Otherwise, the scale factor will be adjusted so that the source image maps to the
+rounded output size.)code", true)
+  .AddOptionalArg<vector<float>>("roi_start", R"code(Origin of the input region of interest (ROI).
+
+Must be specified together with ``roi_end``. The coordinates follow the tensor shape, which is
+the same as size. The coordinates can be either absolute (in pixels, which is the default) or
```

Comment:
The "correction" broke it :(
```suggestion
Must be specified together with ``roi_end``. The coordinates follow the tensor shape order, which is
the same as for ``size`` argument. The coordinates can be either absolute (in pixels, which is the default) or
```

Reply:
Done